src/components/studio-footer/StudioFooter.jsx

```diff
@@ -49,7 +51,7 @@ const StudioFooter = ({
         </Button>
         <div className="col border-top ml-2" />
       </div>
-      <Container size="xl" className="px-4">
+      <Container size={full ? undefined : 'xl'} className="px-4">
```

Comment:
If we [add a `size="full"` option](https://github.com/openedx/paragon/pull/3216#discussion_r1746562375) instead, then this becomes a lot clearer:

```suggestion
      <Container size={full ? 'full': 'xl'} className="px-4">
```

Reply:
Check my comment in the Paragon PR